docs/readmes/proposals/p015_tech_debt_week.md

```diff
@@ -0,0 +1,190 @@
+      1. Otherwise comment on the GH issue - to make clear you are assigned
+   3. See if you understand the GH Issue, prepare questions
+4. Reach out to [Issue Lead](#magma-issue-lead) using the provided contact method
+   1. Check with the lead to ensure you understand the problem
+   2. And the preferred solution approach
+   3. If the Issue lacks detail, do not hesitate to ask
+      1. For an example of the shape of the solution
+      2. To existing committed code that is similar
+5. Create an early Draft PR
+   1. Work towards a draft PR early, post to GH as a [Draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/) and
+      1. Directly notify your [Issue Lead](#magma-issue-lead) over preferred communications
+      2. Post it in Slack [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN)
+      3. You can convert to draft if you mistakenly create full fledge PR (`Still in progress? Convert to draft`)
+   2. Review Timeliness and Tone (for both Draft and Final)
+      1. Expect [Timely review](#issues-lead-response-time)
+      2. With a [Spheherd menality](https://github.com/magma/magma/blob/master/docs/readmes/contributing/contribute_codeowners.md#shepherds-not-gatekeepers)
+      3. Feel free to ask for precise assistance or examples (e.g. a GH gist with example code on an ask)
+   3. The [Issue Lead](#magma-issue-lead) will aim to make contact daily if they haven't heard from you
+   4. Apply any directional changes advised to draft PR
+6. When a final PR is ready for review
+   1. click `Ready for Review` to convert from Draft PR to PR
+   2. Directly notify your [Issue Lead](#magma-issue-lead) over preferred communications
+   3. Post it in Slack [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN)
+   4. See above Review Timeliness and Tone
+7. If Review is completed but you are blocked by CI Checks
+   1. Reach out to your [Issue Lead](#magma-issue-lead) or the [TDW On-Call](#tdw-on-call-contact) for help
+8. When Review is complete and CI Checks are clear
+   1. Reach out to [Issue Lead](#magma-issue-lead) or [TDW On-Call](#tdw-on-call-contact) to Merge
+
+## Magma TDW On-Call
+
+### TDW On-call Contact
+
+- [Video conference](TBD LINK) monitored by [TDW On-Call](#magma-tdw-on-call)
+- Slack channel [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN) monitored by [TDW On-Call](#magma-tdw-on-call)
+- Page: email the TDW On-call at tdw@magma.pagerduty.com
+  - Provide a preferred contact (e.g. slack id, github id, ...)
+
+### TDW On-Call Response time
+
+During TDW, We ask that the following interactions and actions are given `highest work priority` - ideally with a response time of `< 1 hour`.  This is to ensure maximum velocity for our external contributors during this tight week work window - and to ensure we are efficient with their donated time.
```

Comment:
Should we specify time too? like response time <1 hour between 9AM pacific and 3PM pacific?

Reply:
Good point! Done.